docs/cloud/index.md

```diff
@@ -11,6 +11,8 @@ tags:
     - workspaces
     - PaaS
 title: Prefect Cloud
+search:
+  boost: 2
```

Comment:
Do we really need to remember to specify this on every file we write?

Reply:
Is there a way to set a default boost then turn it _down_ in the api reference?

Reply:
Yeah, no parent unfortunately.

I'm assuming that when someone creates a new page they copy the meta-data at the top over from another page (at least I do) - this includes tags and other information. As all of the pages one would copy from would include these two lines, I don't believe that there would be an increased burden. 

The downside if the boost isn't included is relatively low, as the boost value chosen (2) isn't that high.

If the boost value were desired to be changed, the text editor search/replace functionality should make it a low lift.

Reply:
@discdiver what if we downrank the API reference with a fractional boost:

https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#rank-down

```
---
search:
  boost: 0.5
---
```

?

Reply:
We could do that @chrisguidry, but there are even more API reference docs, so went this route.

Reply:
Ohhh right right, gotcha